Tajpuriya grammer - Sheet1.csv

```diff
@@ -0,0 +1,41 @@
+English,Tajpuriya
+Go,जा ,

```

Comment:
I think there is still an extra space and comma here, it should be: `Go,जा` not `Go,जा ,` the newline is parsed as a new entry so we don't need the comma